hana/ha_cluster.sls

```diff
@@ -1,6 +1,23 @@
 {%- from "hana/map.jinja" import hana with context -%}
 {% set host = grains['host'] %}
 
+{% if hana.scale_out %}
+{% set sr_hook_path = '/usr/share/SAPHanaSR-ScaleOut' %}
+remove_SAPHanaSR:
+  pkg.removed:
+    - pkgs:
+      - SAPHanaSR
+      - SAPHanaSR-doc
+
+install_SAPHanaSR_ScaleOut:
+  pkg.installed:
+    - pkgs:
+      - SAPHanaSR-ScaleOut
+      - SAPHanaSR-ScaleOut-doc
+{% else %}
```

Comment:
Let's do one thing. As we started installing the packages for scale out, let's install as well for scale up (the `SAPHanaSR`). This is done now in the `habootstrap-formula`, but to continue with the same consistency, let's install them here. It makes sense

Reply:
Resource-Agents are removed from the pillars for `habootstrap-formula` and installed here now.